src/org/opensolaris/opengrok/analysis/golang/Consts.java

```diff
@@ -0,0 +1,64 @@
+/*Patrick Lundquist
```

Comment:
ok, so actually just one small prob, header has to be proper, please move the name to @author section, this line will break checkstyle ... ;)